src/SignalR/clients/ts/signalr/tests/HttpConnection.test.ts

```diff
@@ -150,6 +150,34 @@ describe("HttpConnection", () => {
         });
     });
 
+    it("cannot send with an un-started connection", async () => {
+        await VerifyLogger.run(async (logger) => {
+            const connection = new HttpConnection("http://tempuri.org");
+
+            await expect(connection.send("LeBron James"))
+                .rejects
+                .toThrow("Cannot send data if the connection is not in the 'Connected' State.");
```

Comment:
Can you add a test for one of the thrown exceptions that get aggregated into a rejected promise?